R/title.R

```diff
@@ -22,6 +23,7 @@
 #' (note that it is not guaranteed to be unique on the server). This behavior
 #' can be disabled by setting `unique = FALSE`.
 #'
+#' @keywords internal
```

Comment:
I've kept this internal because I don't think it's really a user facing function